Cargo.toml

```diff
@@ -28,6 +29,7 @@ thiserror = "1.0"
 bitflags = "1.2"
 lazy_static = "1.4"
 serde = { version = "1.0", features = ["derive"], optional = true }
+image = {version = "0.25.1", optional = true}
```

Comment:
Seems `image` isn't used for anything else other than getting the `RgbaImage` type, so we can add `default-features = false` to avoid pulling in all the other stuff, like jpeg, png, etc decoders.

Reply:
Good idea. I just submitted a commit for it!